### PR TITLE
Issue #240 [Form-Hidden] edit mode UI enhancement.

### LINKED ISSUE #240: [FormHidden] Display field name and value on placeholder
## Feature Request

**Is your feature request related to a problem? Please describe.**
When using multiple Form Hidden Components, there is no way to tell how each one is configured without opening/closing each dialog. We need a way for authors to be able to tell what each Form Hidden Component's configured values are easily.
**Before:**
image
**After:**
image

**Describe the solution you'd like**
Pull Request will be opened with the solution for both v1 and v2 versions

**Are there alternatives?**
N/A

**Documentation**
N/A - Only a change to the UI in edit mode

### PR DESCRIPTION
**Pull request by brandonmaynard**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Fixes #240 
| Patch: Bug Fix?          | No
| Minor: New Feature?      | Yes
| Major: Breaking Change?  | No
| Tests Added + Pass?      | Yes
| Documentation Provided   | No - edit mode UI change only
| Any Dependency Changes?  | No
| License                  | Apache License, Version 2.0

### CODE REVIEW

File: content/src/content/jcr_root/apps/core/wcm/components/commons/v1/templates.html [hunk 1]
```diff
@@ -18,4 +18,10 @@
     <div data-sly-test="${(wcmmode.edit || wcmmode.preview) && isEmpty}"
          class="cq-placeholder ${classAppend}"
          data-emptytext="${component.properties.jcr:title}${emptyTextAppend && ' - '}${emptyTextAppend}"></div>
-</sly>
\ No newline at end of file
+</sly>
+
+<sly data-sly-template.hiddenFieldPlaceholder="${@ isEmpty, classAppend, id, name, value}">
```
Comment (HitmanInWis) [inline content/src/content/jcr_root/apps/core/wcm/components/commons/v1/templates.html]: If a hidden field is always empty, should we also remove `isEmpty` parameter?

Comment (richardhand) [reply content/src/content/jcr_root/apps/core/wcm/components/commons/v1/templates.html]: I think I would prefer if we just use the current placeholder template and make use of the `emptyTextAppend` parameter for building and adding this string in the hidden field itself.

[the remark this answers is not in the document]
Comment (brandonmaynard) [reply content/src/content/jcr_root/apps/core/wcm/components/commons/v1/templates.html]: Hi @richardhand , I have updated the PR to use the regular placeholder template and I also removed the ID so only name and value display. Take a look at:

image

Comment (brandonmaynard) [reply content/src/content/jcr_root/apps/core/wcm/components/commons/v1/templates.html]: This has been addressed, please see comment above.

**Comment by richardhand** [comment]
Works great, thanks @brandonmaynard for the contribution.

**Pull request merged**